### PR TITLE
Fix broken link in user_manual/groupware/sync_osx.

### PR DESCRIPTION
**Pull request by camilasan**
Add new label to refer to section in another page.

### CODE REVIEW

**Review by kesselb — APPROVED** [review]

File: user_manual/groupware/sync_osx.rst [hunk 1]
```diff
@@ -26,7 +26,7 @@ and **CardDAV** (Contacts) to your Nextcloud.
 
    **Username**: Your Nextcloud username or email
 
-   **Password**: Your generated app-password/token (`Learn more <https://docs.nextcloud.com/server/latest/user_manual/session_management.html#managing-devices>`_).
+   **Password**: Your generated app-password/token (:ref:`Learn more<managing_devices>`).
 
 .. TODO ON RELEASE: Update version number above on release
```
Comment (nickvergessen) [inline user_manual/groupware/sync_osx.rst:31]:
```suggestion
```

It's not version dependent now anymore

Comment (kesselb) [reply user_manual/groupware/sync_osx.rst]: Well spotted :+1:

File: user_manual/groupware/sync_osx.rst [hunk 2]
```diff
@@ -26,15 +26,13 @@ and **CardDAV** (Contacts) to your Nextcloud.
...
-
-.. TODO ON RELEASE: Update version number above on release
+   **Password**: Your generated app-password/token (:ref:`Learn more<managing_devices>`).
 
    **Server Address**: URL of your Nextcloud server (e.g. ``https://cloud.example.com``)
 
 .. figure:: ./images/macos_3.png
 
-5. Click on **Sign In**.
+1. Click on **Sign In**.
```
Comment (nickvergessen) [inline user_manual/groupware/sync_osx.rst:35]: ??

**Pull request merged**